docs/assets/main.css

```diff
@@ -22978,6 +22978,8 @@ body {
       background-color: #F7F9FA; }
     .apilist__listing a {
       text-decoration: none !important; }
+    .apilist__listing figure img, figure a img {
+      height: 40px; }
```

Comment:
I would replace this with:
`.apilist__listing figure img { max-height: 40px; }` to apply only on that view and constraint height whether they exceed.

Reply:
As you wish :)